### PR TITLE
Added Arabic translation [ar-YE]

### PR DESCRIPTION
**Pull request by HasanAlyazidi**
## PR Type
What kind of change does this PR introduce?

Translation

### Main:
Main

### Settings:
Settings

## Other information
There is a bug that appear only for RTL languages and I will create an issue for that.

### CODE REVIEW

File: src/Notepads/Notepads.csproj [hunk 1]
```diff
@@ -449,6 +449,15 @@
   <ItemGroup>
     <None Include="Package.targets" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (JasonStein) [inline src/Notepads/Notepads.csproj:452]: You can remove these lines since it will be covered by <PRIResource Include="Strings\**\*.resw" />

**Review by JasonStein — CHANGES_REQUESTED** [review]

File: src/Notepads/Notepads.csproj (hunk 1 above)
Comment (HasanAlyazidi) [reply src/Notepads/Notepads.csproj:452]: Only L449 to L451?

Comment (soumyamahunt) [reply src/Notepads/Notepads.csproj:452]: @HasanAlyazidi only remove the lines you have added L452 to L460.

Comment (HasanAlyazidi) [reply src/Notepads/Notepads.csproj]: @soumyamahunt @JasonStein done.

**Pull request merged**